api/v1/controllers/UserController.js

```diff
@@ -90,7 +90,8 @@ function requestPasswordReset(req, res, next) {
 router.use(bodyParser.json());
 router.use(middleware.auth);
 
-router.post('/', middleware.request(requests.BasicAuthRequest), createUser);
+router.post('/', middleware.request(requests.BasicAuthRequest),
+  middleware.permission(roles.ORGANIZERS), createUser);
```

Comment:
we we're leaning towards it two weeks ago, but we can't probably discuss it more next meeting